### DISCUSSION TITLE
parent beta grain reconstruction/ point group

### DISCUSSION THREAD

**Discussion by antonthiel** [mtex-toolbox/mtex#951] [Ask Anything] [answered]
Hello Ralf, 
ive got a question regarding to the  default setting of the point group of Alpha Titanium. Why do you have choosen 622 instead of 6/mmm or which benefits has it?  I dont understand it, because it is a pure metall and therefore there arent specific atom types at specific positions., which would requires 622.
Is it also possible to use the script of the parent beta grain reconstruction with 6/mmm instead of 622?

Thanks a lot for this great tool you've created! It's very efficient and smart!

**Comment by ralfHielscher** [comment, marked as answer]
There is no particular reason to work with 622 instead of 6/mmm. Reconstruction will be exactly the same for both settings.

Ralf.
